scripts/sct_propseg.py

```diff
@@ -10,16 +10,16 @@
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
-from msct_parser import Parser
-import sys
-import sct_utils as sct
 import os
 import shutil
-from scipy import ndimage as ndi
+import sys
+
 import numpy as np
-from sct_image import orientation
 import sct_image
-
+import sct_utils as sct
+from msct_parser import Parser
+from scipy import ndimage as ndi
```

Comment:
Preferred order is stdlib, 3rd party, then internal; scipy is 3rd party and should go after numpy.

Reply:
I am using "isort", auto-sort option generates this. I belive it might be because of importing from "class-containing module" convention.